apps/web/src/app/home/components/my-open-pull-requests/my-open-pull-requests.tsx

```
@@ -60,45 +65,61 @@ export const MyOpenPullRequests = (props: BoxProps) => {
+    <Stack>
+      <Divider
+        label={
+          <>
+            <Group gap={5} align="center">
+              🫵 My open work
+              <Badge variant="default" size="xs">
+                {pullRequests?.length || 0}
+              </Badge>
+            </Group>
+          </>
+        }
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)